src/app/_components/ListboxButton.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-import { Listbox } from '@headlessui/react'
+import { ListboxButton as HeadlessListboxButton } from '@headlessui/react'
```

Comment:
I would be specific with naming.

```suggestion
import { ListboxButton as HeadlessUIListboxButton } from '@headlessui/react'
```